Scripts/Swift/SetUpCICDService.swift

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/swift
+
+import Foundation
+
+let fileManager = FileManager.default
+
+print("Which CI/CD service do you use (Can be edited later) [(g)ithub/(b)itrise/(c)odemagic/(l)ater]: ")
+
+var service = "later"
+
+enum CICDService {
+    case github, bitrise, codemagic, later
+
+    init(_ name: String) {
+        switch name {
+        case "g", "github", "Github":
+            self = .github
+        case "b", "bitrise", "Bitrise":
+            self = .bitrise
+        case "c", "codemagic", "Codemagic":
+            self = .codemagic
+        default: self = .later
+        }
+    }
+}
+
+service = readLine() ?? service
```

Comment:
I'd like to suggest two things 🙏 
- Instead of letting the user config the CICD service later as a `later` option, consider "Hit Enter if you want to edit/update later". We, developers, don't have to add a not-so-meaningful enum case. The users don't have to remember one more option–they might rerun this for the same or multiple projects, and easily to move forward.
- When user faces the `g`, `b`, and `c` options, they might select the unwanted option. For e.g, they want to choose `g` for GitHub but hit the wrong key on the keyboard, say `v` (a key nearby). In that case, the script process is as if the user selects `later`. It would confuse users. IMO we should try to ensure users select what they want, not providing a fallback. A simple and widely-used solution is to run a loop around `readLine()`.

Reply:
Added while loop to this method. Here's the demo 
<img width="1136" alt="Screen Shot 2023-07-17 at 17 53 45" src="https://github.com/nimblehq/ios-templates/assets/6356137/e84a7f25-6568-4786-a1f7-de2ee931852f">
I didn't added the while loop in `DeliveryConstants` because that config has a lower significance than this config 🙏